src/Modules/Fields/EditMeshBoundingBox.h

```diff
@@ -92,16 +99,35 @@ namespace SCIRun {
         MODULE_TRAITS_AND_INFO(ModuleHasUI)
 
       private:
-        void executeImpl(FieldHandle f);
-        void clear_vals();
-        void update_input_attributes(FieldHandle);
-        void computeWidgetBox(const Core::Geometry::BBox& box) const;
-        Graphics::Datatypes::GeometryHandle buildGeometryObject();
+        void clearVals();
+        void computeWidgetBox(const Core::Geometry::BBox& box);
+        void buildGeometryObject();
+        void updateState(FieldHandle field);
+        void sendOutputPorts();
+        void resetToInputField();
         void processWidgetFeedback(const Core::Datatypes::ModuleFeedback& var);
-        void adjustGeometryFromTransform(const Core::Geometry::Transform& transformMatrix);
-
-        boost::shared_ptr<EditMeshBoundingBoxImpl> impl_;
-        bool widgetMoved_;
+        void adjustGeometryFromTransform(Core::Datatypes::MouseButton btn,
+                                         Core::Datatypes::WidgetMovement move,
+                                         const Core::Geometry::Transform& feedbackTrans);
+        void generateGeomsList();
+        void saveToParameters();
+        void loadFromParameters();
+        std::string convertForLabel(double coord);
+        void updateInputFieldAttributes();
+
+        Core::Geometry::Point mPos;
```

Comment:
Change all these names. SCIRun naming standard (except for spire--nonstandard code) is `instanceVar_` trailing underscore